### PR TITLE
Add Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by mathessaricci**

### CODE REVIEW

**Comment by mathessaricci** [comment]
Here I am again, trying to translating some of your apps to my beloved language. This time I think I won't make the same mistakes. HAHAHAHHA

File: po/extra/pt_BR.po [hunk 1]
```diff
@@ -0,0 +1,456 @@
...
+#: data/reco.appdata.xml.in:72
+#, fuzzy
+msgid "Update Portuguese translation (thanks to @hugok79)"
+msgstr "Tradução portuguesa adicionada (obrigado a @hugok79)"
+
+#: data/reco.appdata.xml.in:73
+#, fuzzy
+msgid ""
+"Update Spanish translation and add Catalan translation (thanks to @fitojb)"
+msgstr "Atualização da tradução e adicionado Catalão (obrigado a @CCaroV)"
```
Comment (ryonakano) [inline po/extra/pt_BR.po:105]:
```suggestion
msgid ""
"Update Spanish translation and add Catalan translation (thanks to @fitojb)"
msgstr "Atualização da tradução e adicionado Catalão (obrigado a @fitojb)"
```

File: po/pt_BR.po [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+msgstr "Sistema"
+
+#: src/Views/WelcomeView.vala:43
+msgid "Both"
+msgstr "Ambos"
+
+#: src/Views/WelcomeView.vala:45
+#, fuzzy
+msgid "Microphone to use:"
+msgstr "Microfone"
```
Comment (ryonakano) [inline po/pt_BR.po:76]: * Remove the "fuzzy" flag
* Correct translation

File: po/extra/pt_BR.po [hunk 3]
```diff
@@ -0,0 +1,456 @@
...
+msgstr "Tela de gravação"
+
+#: data/reco.appdata.xml.in:33
+msgid "Recording is successfully finished"
+msgstr "A gravação foi concluída com sucesso"
+
+#: data/reco.appdata.xml.in:72
+#, fuzzy
+msgid "Update Portuguese translation (thanks to @hugok79)"
+msgstr "Tradução portuguesa adicionada (obrigado a @hugok79)"
```
Comment (ryonakano) [inline po/extra/pt_BR.po:99]: * Remove the "fuzzy" flag
* The word "Update" seems to be translated as "Add" maybe? (as far as I checked this translation using Google Translate)

File: po/pt_BR.po [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+#: src/MainWindow.vala:165
+msgid "Save"
+msgstr "Salvar"
+
+#: src/MainWindow.vala:165
+msgid "Cancel"
+msgstr "Cancelar"
+
+#~ msgid "Computer"
+#~ msgstr "Computador"
```
Comment (ryonakano) [inline po/pt_BR.po:199]:
```suggestion
```

This is unused so let's remove it

**Review by ryonakano — CHANGES_REQUESTED** [review]
Hi, thanks again!

I guess you copied `pt.po` files and updated to Brazilian Portuguese. That's fine, but you need to remove the `#, fuzzy" flag and check if that translation gets old against the current English string.

**Comment by ryonakano** [comment]
BTW here is how to edit your Pull Requests without closing.

If you perform changes request from reviewers without modification, click **"Commit suggestion"** and then push the green button labeled **"Commit changes"**.

Screenshot from 2021-08-20 06-26-34

If you want to edit other part of a file that I reviewed, you can go to the **"Files changed"** tab and click **"Edit file"**:

Screenshot from 2021-08-20 06-26-55

**Pull request closed without merging**